socketio/src/payload.rs

```diff
@@ -52,6 +72,8 @@ impl AsRef<[u8]> for Payload {
         match self {
             Payload::Binary(b) => b.as_ref(),
             Payload::String(s) => s.as_ref(),
+            // TODO: how to handle this case
+            Payload::StringArray(_a) => &[0u8; 0],
```

Comment:
Mh... sounds like we would need to drop this impl. I don't see any way this would make sense...

Reply:
yeah was kinda confused where this would be used, and dropping it will make life easier